packages/react-components/react-skeleton/src/components/Skeleton/useSkeletonContextValues.ts

```diff
@@ -0,0 +1,12 @@
+import { SkeletonContextValue, SkeletonContextValues, SkeletonState } from '../Skeleton';
+
+export const useSkeletonContextValues = (state: SkeletonState): SkeletonContextValues => {
+  const { animation, appearance } = state;
+
+  const skeletonGroup: SkeletonContextValue = {
+    animation,
+    appearance,
+  };
```

Comment:
You're using a usual `React.createContext()` (and this is totally fine!), but we need to memoize a context value to avoid needless re-renders:

```suggestion
  const skeletonGroup = React.useMemo(() => ({
    animation,
    appearance,
  }), [animation, appearance]);
```